torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1631,6 +1631,8 @@ def _init(
         self.fx_node_cache: Dict[Tuple[Callable, Tuple[Any, ...]], torch.fx.Node] = {}
         self.source_to_symbol: Dict[str, sympy.Symbol] = {}
 
+        self.source_to_symint_node_cache : Dict[TensorPropertySource, SymInt] = {}
```

Comment:
Probably best to put this into its own PR

Reply:
That's fine by me